WIP 3.0 Chapters/Chapter 6 Spells

```diff
@@ -1882,7 +1882,7 @@ ___
 - **Components:** V, S, M (a holy symbol)
 - **Duration:** Instantaneous
 ___
-You grasp the fabric around a target within range, pulling necromantic energies from it and creatures of your choice within 5 feet of it. A target must make a Constitution saving throw, taking 2d4 necrotic damage on a failed save, or half as much damage on a successful one.
+You pull on the veil separating reality and the realm the void realm, creating small fissures where its maddening energies leak through. A creature within range and creatures of your choice within 5 feet of it, must succeed on a Wisdom saving throw or take 2d4 psychic damage on a failed save, or half as much damage on a successful one.
```

Comment:
> veil separating reality and the realm the void realm
I think there is on "the realm" to much but that's on me because you just copied my suggestion 😅